package.json

```diff
@@ -19,6 +19,7 @@
   "scripts": {
     "build": "yarn workspaces run build",
     "build:dev": "yarn workspaces run build:dev",
+    "ci:pr-checks": "./.bin/pr-checks",
```

Comment:
I believe it would need to be part of `node_modules` for it to be accessible directly vs putting it under `scripts`. 

The only ones I see for execution come up when I run `yarn run`:
```
@sharky.corp.adobe.com ➜ pwa-studio git:(aws-cicd) yarn run
yarn run v1.15.2
info Commands available from binary scripts: JSONStream, acorn, ansi-html, apollo-codegen, atob, autoprefixer, babel, babel-external-helpers, babylon, browserslist, build-storybook, bundlesize, bundlesize-init, bundlesize-pipe, concurrently, conventional-changelog-writer, conventional-commits-parser, conventional-recommended-bump, coveralls, cssesc, danger, danger-ci, danger-init, danger-js, danger-local, danger-pr, danger-process, danger-reset-status, danger-runner, detect, detect-libc, detect-port, disparity, errno, escodegen, esgenerate, eslint, esparse, esvalidate, get-pkg-repo, git-raw-commits, git-semver-tags, gql, graphql, graphql-schema-linter, handlebars, he, html-minifier, hulk, husky-upgrade, import-local-fixture, is-ci, jest, jest-runtime, js-yaml, jsesc, jsome, json5, lerna, loose-envify, miller-rabin, mime, multicast-dns, nearley-railroad, nearley-test, nearley-unparse, nearleyc, needle, node-gyp, node-pre-gyp, nopt, npm-path, npm-run, npm-tree, npm-which, opener, parse-github-url, parse-nodejs-lockfile, parser, pbjs, pbts, pjv, prebuild-install, prettier, prettier-check, rc, react-docgen, regexp-tree, regjsparser, rimraf, run-node, sane, semver, sha.js, shjs, sl-log-transformer, snyk, sshpk-conv, sshpk-sign, sshpk-verify, start-storybook, storybook-server, strip-indent, svgo, swagger-to-graphql, tap-diff, tap-parser, tap-xunit, tape, terser, tree-kill, ts-node, txunit, uglifyjs, upward-js-server, upward-spec, uuid, watch, webpack, webpack-bundle-analyzer, webpack-cli, webpack-dev-server, which, window-size, z-schema, zopfli, zopflipng
```